tabular/src/autogluon/tabular/models/knn/knn_model.py

```diff
@@ -39,14 +39,14 @@ def _set_default_params(self):
         for param, val in default_params.items():
             self._set_default_param_value(param, val)
 
-    def _set_default_auxiliary_params(self):
-        default_auxiliary_params = dict(
+    def _get_default_auxiliary_params(self) -> dict:
+        default_auxiliary_params = super()._get_default_auxiliary_params()
+        extra_auxiliary_params = dict(
             ignored_type_group_raw=[R_CATEGORY, R_OBJECT],  # TODO: Eventually use category features
             ignored_type_group_special=[S_TEXT_NGRAM, S_TEXT_SPECIAL, S_DATETIME_AS_INT],
```

Comment:
why does KNN need to ignore S_DATETIME_AS_INT? Because of lack of feature-rescaling?

Reply:
Yes, currently the datetime as int is a massive integer that would overpower all the other features.